api/src/test/java/org/eclipse/microprofile/rest/client/ext/MockConfigProviderResolver.java

```diff
@@ -78,14 +81,60 @@ public String getValue(String propertyName) {
                     public String getName() {
                         return "MockConfigSource";
                     }
+
+                    @Override
+                    public Set<String> getPropertyNames() {
+                        return getSystemProps().keySet();
+                    }
                 });
             }
 
-            @SuppressWarnings({"unchecked", "rawtypes"})
+            @SuppressWarnings({ "unchecked", "rawtypes" })
             private Map<String, String> getSystemProps() {
                 Map sysProps = (Map) System.getProperties();
                 return (Map<String, String>) sysProps;
             }
+
+            @Override
+            public ConfigValue getConfigValue(String propertyName) {
+                return new ConfigValue(){
+
+                    @Override
+                    public String getName() {
+                        return propertyName;
+                    }
+
+                    @Override
+                    public String getValue() {
+                        return getSystemProps().get(propertyName);
+                    }
+
+                    @Override
+                    public String getRawValue() {
+                        return getSystemProps().get(propertyName);
+                    }
+
+                    @Override
+                    public String getSourceName() {
+                        return "SystemPropWrapper";
```

Comment:
This should be getConfigSources().get(0).getName();

Reply:
`getConfigSources()` returns an `Iterable` instead of a list, so I can use:
`return getConfigSources().iterator().next().getName();`